x-pack/plugins/maps/public/kibana_services.js

```diff
@@ -139,6 +139,10 @@ let dataSearchService;
 export const setSearchService = (searchService) => (dataSearchService = searchService);
 export const getSearchService = () => dataSearchService;
 
+let dataQueryService;
```

Comment:
Instead of exposing queryService like this, can you just use the existing `getData` and then just access queryService like `getData().query`